### PR TITLE
281 create player views

### PR DESCRIPTION
**Pull request by sakarisalmi**
I've created three different player views for retrieving the serialized data of players.
get all players of all matches: /players/
get a specific player by player_id: /players/<int:player_id>
get players of a specific match: /matches/<int:match_id>/players/

tested, works

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -68,6 +72,11 @@
 	path('match/', MatchView.as_view()),
 	path('matches/', MatchHistory.as_view()),
 
+	# Player views
+	path('players/', PlayerListView.as_view()),
+	path('players/<int:player_id>', PlayerDetailView.as_view()),
```
Comment (dorianjagusch) [inline backend/backend/urls.py:77]: What happened to the PR that was fixing those?

Comment (dorianjagusch) [inline backend/backend/urls.py:77]: What happened to the PR that was fixing those?

**Review by dorianjagusch — APPROVED** [review]
Noice

**Review by merituulie — APPROVED** [review]

File: backend/backend/urls.py (hunk 1 above)
Comment (sakarisalmi) [reply backend/backend/urls.py:77]: what do you mean? Fix what?

**Pull request merged**